Web/Models/Checkout/ChargeCreditCard.cs

```diff
@@ -0,0 +1,119 @@
+﻿using AuthorizeNet.Api.Contracts.V1;
+using AuthorizeNet.Api.Controllers;
+using AuthorizeNet.Api.Controllers.Bases;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Web.Models.Checkout
+{
+    public class ChargeCreditCard
+    {
+        private static IConfiguration Configuration { get; }
+        
+        public static ANetApiResponse Run(decimal amount)
+        {
+            Console.WriteLine("Charge Credit Card Sample");
+
+            ApiOperationBase<ANetApiRequest, ANetApiResponse>.RunEnvironment = AuthorizeNet.Environment.SANDBOX;
+
+            // define the merchant information (authentication / transaction id)
+            ApiOperationBase<ANetApiRequest, ANetApiResponse>.MerchantAuthentication = new merchantAuthenticationType()
+            {
+                name = Configuration.GetConnectionString("API_LOGIN_ID"),
+                ItemElementName = ItemChoiceType.transactionKey,
+                Item = Configuration.GetConnectionString("TRANSACTION_KEY"),
+            };
+
+            var creditCard = new creditCardType
+            {
+                cardNumber = "4111111111111111",
+                expirationDate = "1028",
+                cardCode = "123"
```

Comment:
These would probably be inputs to the method rather than hard-coded. Maybe use a `CreditCardPayment` model?